hasktorch/src/Torch/Distributions/Bernoulli.hs

```diff
@@ -33,7 +33,7 @@ instance Distribution Bernoulli where
     sample d = D.bernoulliIO' . F.expand (probs d) False . extendedShape d
     logProb d value = F.mulScalar (-1 :: Int) (bce' (logits d) value)
     entropy d = bce' (logits d) $ probs d
-    enumerateSupport d doExpand = 
+    enumerateSupport d doExpand =
         (if doExpand then \t -> F.expand t False ([-1] <> batchShape d) else id) values
         where
             values = D.reshape ([-1] <> replicate (length $ batchShape d) 1) $ D.asTensor [0.0, 1.0 :: Float]
```

Comment:
so far so good :) happy to try this on more!

Reply:
lots more files incoming!!

Reply:
You’re on your way to becoming the stuff of folklore and legends, your soul being forever immortalized.